### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by hedengran**
Found some broken links in various places while building another documentation update, and this PR fixes those links.

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala [hunk 1]
```diff
@@ -73,7 +73,7 @@ class FulltextIndexTest extends DocumentingTest {
           |While a composite index applies only to entities that match the indexed label and _all_ of the indexed properties, full-text index will index entities that have at least one of the indexed labels or relationship types, and at least one of the indexed properties.
           |"""
       )
-      p("For information on how to configure full-text indexes, refer to <<operations-manual#index-configuration-fulltext-search,  Operations Manual -> Indexes to support full-text search>>.")
+      p("For information on how to configure full-text indexes, refer to <<operations-manual#index-configuration-fulltext,  Operations Manual -> Indexes to support full-text search>>.")
```
Comment (Lojjs) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala:76]: This link seems to be broken in 4.0 and 4.1 documentation as well: https://neo4j.com/docs/cypher-manual/4.1/administration/indexes-for-full-text-search/.

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala [hunk 2]
```diff
@@ -43,7 +43,7 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
       p(
         """In this document we will often use the terms _'allows'_ and _'enables'_ in seemingly identical ways. However, there is a subtle difference.
           |We will use _'enables'_ to refer to the consequences of <<administration-security-reads, read privileges>> where a restriction will not cause an error,
-          |only a reduction in the apparent graph size. We will use _'allows'_ to refer to the consequence of <<administration-security-write, write privileges>>
+          |only a reduction in the apparent graph size. We will use _'allows'_ to refer to the consequence of <<administration-security-writes, write privileges>>
```
Comment (Lojjs) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala:46]: This link seems to be broken in 4.1 documentation as well: https://neo4j.com/docs/cypher-manual/4.1/administration/security/subgraph/#

**Review by Lojjs — APPROVED** [review]
There is something broken on 4.x docs right now so I couldn't build it, but it looks good. One of the broken links are broken in 4.1 as well and one in both 4.0 and 4.1, see comments. It would be amazing if you could back-port those changes!

**Comment by hedengran** [comment]
> There is something broken on 4.x docs right now so I couldn't build it, but it looks good. One of the broken links are broken in 4.1 as well and one in both 4.0 and 4.1, see comments. It would be amazing if you could back-port those changes!

Sure!

**Pull request merged**